.github/workflows/node.js.yml

```diff
@@ -0,0 +1,24 @@
+name: Node CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.12.0, 10.x, 12.x, 14.x]
```

Comment:
I don't know why in the `.travis.yml` there was `10.12.0` but is that so different with `10.x` ?
Maybe we don't need it, and we could only run in `10.x`.
so `node-version: [10.x, 12.x, 14.x]`.

Reply:
I also don't find pinning to 10.12.0 very useful, so I've removed the pinned version.
The tests still run on 10.x anyways.